go.mod

```diff
@@ -3,10 +3,9 @@ module github.com/palomachain/paloma
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.30.0
```

Comment:
so revert these changes the `go get github.com/newrelic/go-agent/v3` did? @verabehr

Reply:
Is it required to downgrade? If so, we may need to look further for an alternative solution, but if it is not required, we definitely don't want to downgrade any of these libraries. They're all core libraries.